cmd/livepeer/livepeer.go

```diff
@@ -54,11 +54,11 @@ func main() {
 		glog.Fatalf("Cannot find current user: %v", err)
 	}
 
-	httpPort := flag.String("http", "8935", "http port")
 	rtmpPort := flag.String("rtmp", "1935", "rtmp port")
 	datadir := flag.String("datadir", fmt.Sprintf("%v/.lpData", usr.HomeDir), "data directory")
-	rtmpIP := flag.String("rtmpIP", "127.0.0.1", "IP to bind for HTTP RPC commands")
-	httpIP := flag.String("httpIP", "127.0.0.1", "IP to bind for HTTP RPC commands")
+	rtmpIP := flag.String("rtmpIP", "127.0.0.1", "IP to bind for RTMP commands")
```

Comment:
For consistency sake, should we combine `rtmpIP` and `rtmpPort` into `rtmpAddr`?  (same as `cliAddr`)

Reply:
Yep, done.

Reply:
https://github.com/livepeer/go-livepeer/pull/507/commits/0d71cbb119561d8ec89227098b60af9bfca629db